### PR TITLE
Feature/fragment builder

### PR DESCRIPTION
**Pull request by RomanTcv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -252,6 +254,67 @@ AppOmegaIntentBuilder.from(this)
...
+    @OmegaExtraModel(prefix = "model")
+    Model model = new Model();
+
+    public TestService() {
+        this(TAG);
+    }
+
+    public TestService(String name) {
+        super(name);
+    }
```
Comment (anton-knyazev) [inline README.md]: один конструтор

File: core/src/main/java/com/omega_r/libs/omegaintentbuilder/bundle/BundleBuilder.kt [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+
+    private val bundle = Bundle()
+
+    /**
+     * Inserts all mappings from the given Bundle into this Bundle.
+     *
+     * @param bundle a Bundle
+     * @return This BundleBuilder for method chaining
+     */
+    fun putExtra(bundle: Bundle): BundleBuilder {
```
Comment (anton-knyazev) [inline core/src/main/java/com/omega_r/libs/omegaintentbuilder/bundle/BundleBuilder.kt]: fun putExtra(name: String, bundle: Bundle): BundleBuilder

File: examples/src/main/java/com/omega_r/omegaintentbuilder/MainActivity.java [hunk 3]
```diff
@@ -230,4 +235,12 @@ private void onExtrasToServiceClicked() {
                 .startService();
     }
 
+
+    private void onExtrasToFragmentClicked() {
+        AppOmegaIntentBuilder.from(this)
+                .appActivity()
```
Comment (anton-knyazev) [inline examples/src/main/java/com/omega_r/omegaintentbuilder/MainActivity.java]: appActivities

**Review by anton-knyazev — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (RomanTcv) [reply README.md]: Caused by: java.lang.InstantiationException: java.lang.Class<com.omega_r.omegaintentbuilder.service_example.TestService> has no zero argument constructor

**Review by anton-knyazev — APPROVED** [review]

**Pull request merged**